Makefile

```diff
@@ -3,12 +3,22 @@
 # vim: set fileencoding=utf-8:
 
 
-DOCKER_IMAGE=$(shell cat dockerimagename.txt)
-DOCKER_VERSION=$(shell cat dockerimageversion.txt)
+DOCKER_IMAGE_NAME_FILE=dockerimagename.txt
+DOCKER_IMAGE_VERSION_FILE=dockerimageversion.txt
+DOCKER_IMAGE=$(shell cat $(DOCKER_IMAGE_NAME_FILE))
+DOCKER_VERSION=$(shell cat $(DOCKER_IMAGE_VERSION_FILE))
 
 include ./build.mk
 
+
+version:
+	cp "$(DOCKER_IMAGE_VERSION_FILE)" arm64
```

Comment:
Still not sure what the reason for all of this duplication for arm64 supporting arm64 platform

Reply:
There may be substantial differences in the underlying Jupyter Datascience Notebook when it happens because of the R and Lua dependencies on just-in-time C and FORTRAN compilers vs the x86 versions.  That's the reason why the Jupyter team hasn't released it on ARM yet.  Some of those differences may/will percolate to how we build the Lucyfer ARM images.  We can't tell until a Jupyter Datascience Notebook ARM is available.